app/views/about.html

```diff
@@ -16,11 +16,17 @@ <h2>About</h2>
                 <h2 id="version">Version</h2>
                 <dl class="dl-horizontal left">
                   <dt>OpenShift Master:</dt>
-                  <dd>{{version.master.openshift || 'unknown'}}</dd>
+                  <dd>
+                    {{version.master.openshift.gitVersion}}
```

Comment:
we officially documented `version.master.openshift` and `version.master.kubernetes` in the about page customization.  I'd rather not change those scope variables unnecessarily.